### PR TITLE
✨ [Amp story] Scaffold desktop one panel experiment

### PR DESCRIPTION
**Pull request by processprocess**
Scaffolds files and tests for `amp-story-desktop-one-panel` experiment.

- New CSS class on story element
- New CSS file
- Template with experiment toggle
- Basic visual test
- Basic unit test

### CODE REVIEW

File: extensions/amp-story/1.0/amp-story-store-service.js [hunk 1]
```diff
@@ -54,6 +54,7 @@ export const UIType = {
   DESKTOP_PANELS: 1, // Default desktop UI.
   DESKTOP_FULLBLEED: 2, // Desktop UI if landscape mode is enabled.
   VERTICAL: 3, // Vertical scrolling versions, for search engine bots indexing.
+  DESKTOP_ONE_PANEL: 4, // Experiment for new default desktop UI.
```
Comment (gmajoulet) [inline extensions/amp-story/1.0/amp-story-store-service.js:57]: Nits:
- a-z
- plz use a comment that describes the experience more than the state of the project. Feel free to rewrite the `DESKTOP_PANELS` comment if that helps too

File: extensions/amp-story/1.0/amp-story.js [hunk 2]
```diff
@@ -1814,6 +1815,7 @@ export class AmpStory extends AMP.BaseElement {
           this.element.removeAttribute('desktop');
           this.element.classList.remove('i-amphtml-story-desktop-panels');
           this.element.classList.remove('i-amphtml-story-desktop-fullbleed');
+          this.element.classList.remove('i-amphtml-story-desktop-one-panel');
```
Comment (gmajoulet) [inline extensions/amp-story/1.0/amp-story.js:1818]: Plz also remove this class when switching to another UI type, that could help with your "toggle experiment" button when you're on desktop
The only one we don't really care about is vertical.

**Review by gmajoulet — APPROVED** [review]

**Comment by processprocess** [comment]
cc @jridgewell for OWNERS approval on `config.js`.

**Review by danielrozenberg — APPROVED** [review]
Approval for build-system/test-configs/config.js

**Review by jridgewell — APPROVED** [review]

**Pull request merged**